test/ngAnimate/animateCssDriverSpec.js

```diff
@@ -104,6 +104,14 @@ describe("ngAnimate $$animateCssDriver", function() {
         driver({ element: element });
         expect(capturedAnimation[1].applyClassesEarly).toBeFalsy();
       }));
+
+      it("should not provide a `method` as an option value if the animation is not structural", inject(function() {
```

Comment:
I think we could simplify this by changing it "should only provide ... if the animation is structural". What still confuses me, is that "method"  doesn't really reflect what is set. It's really the event (name) that is set, right?